### PR TITLE
initial draft of cloud_cli guide

### PR DESCRIPTION
**Pull request by coapacetic**
## What are you changing in this pull request and why?
This is a net new addition to the docs repo to create a guide for using dbt Cloud CLI!  The initial motivation here is to give Coalesce workshop attendees and easy way to access links and code snippets for the workshop.  Post coalesce, we can convert this into an evergreen guide that we keep updated with the latest and greatest for dbt Cloud CLI.

## Checklist
- [ ] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."
- [ ] Add and Review all images
- [ ] Review all copy for placeholders
- [ ] Add and review metadata and logos

### CODE REVIEW

**Review by ramiz-bozai — APPROVED** [review]
Looks good to me, made direct edits to Deferral , Linting and Mesh sections

File: website/docs/guides/qs-cloud-cli.md [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+
+1. In dbt Cloud, select Platform Analytics and choose **Develop >> Configure Cloud CLI**.
+
+1. Based on your current local setup, use the following guidance to determine your installation approach:
+
+    a. Check if you have dbt in your PATH by running `dbt --version`
+
+    b. If you don't have dbt in your PATH, we recommend the macOS or Windows installation method.
+
+    c. If you do have dbt in your PATH (global environment) we recommend i) uninstalling dbt globally and ii) installing dbt Cloud CLI with a python virtual environment.
```
Comment (nghi-ly) [inline website/docs/guides/qs-cloud-cli.md]:
```suggestion
    c. If you do have dbt in your PATH (global environment) we recommend i) uninstalling dbt globally and ii) installing dbt Cloud CLI with a Python virtual environment.
```

File: website/docs/guides/qs-cloud-cli.md [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+
+1. Based on your current local setup, use the following guidance to determine your installation approach:
+
+    a. Check if you have dbt in your PATH by running `dbt --version`
+
+    b. If you don't have dbt in your PATH, we recommend the macOS or Windows installation method.
+
+    c. If you do have dbt in your PATH (global environment) we recommend i) uninstalling dbt globally and ii) installing dbt Cloud CLI with a Python virtual environment.
+
+    d. If you have dbt in a virtual environment, install dbt Cloud CLI with a separate python virtual environment. Be sure to activate it with `source <path to env>/bin/activate`.
```
Comment (nghi-ly) [inline website/docs/guides/qs-cloud-cli.md]:
```suggestion
    d. If you have dbt in a virtual environment, install dbt Cloud CLI with a separate Python virtual environment. Be sure to activate it with `source <path to env>/bin/activate`.
```

File: website/docs/guides/qs-cloud-cli.md [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+
+1. Navigate to the dbt project folder that you cloned earlier and open the `dbt_project.yml` file with your `project_id`.
+
+### Confirm the installation
+
+Run `dbt compile` to verify your installation.
+
+There you go! You've installed the dbt Cloud CLI! Let's dive in!
+
+### Additional Resources
```
Comment (nghi-ly) [inline website/docs/guides/qs-cloud-cli.md]:
```suggestion
### Additional resources
```

File: website/docs/guides/qs-cloud-cli.md [hunk 4]
```diff
@@ -0,0 +1,296 @@
...
+1. Navigate to the dbt project folder that you cloned earlier and open the `dbt_project.yml` file with your `project_id`.
+
+### Confirm the installation
+
+Run `dbt compile` to verify your installation.
+
+There you go! You've installed the dbt Cloud CLI! Let's dive in!
+
+### Additional resources
+Consult the docs below if you run into specific blocker related to dbt Cloud CLI installation:
```
Comment (nghi-ly) [inline website/docs/guides/qs-cloud-cli.md]:
```suggestion
Consult the following docs if you run into problems when trying to install the dbt Cloud CLI:
```

File: website/docs/guides/qs-cloud-cli.md [hunk 5]
```diff
@@ -0,0 +1,296 @@
...
+* `dbt compile --select stg_campaigns` &mdash; Compiles your dbt project
+* `dbt run --select stg_campaigns` &mdash; Materialized your dbt models
+* `dbt run --select stg_campaigns` &mdash; Preview the results of a model
+* `dbt test --select stg_campaigns` &mdash; Execute tests against your materialized models
+
+Now let's dive into some more advanced components of dbt Cloud CLI.
+
+### Deferral
+
+Deferral is a powerful functionality that lets users leverage upstream assets that exist outside of a user's personal development environment. As a result user's can speed up their development workflows and save on warehouse compute. Let's run a few commands using deferral:
```
Comment (nghi-ly) [inline website/docs/guides/qs-cloud-cli.md]:
```suggestion
Deferral is a powerful functionality, allowing you to leverage upstream assets that exist outside of your personal development environment. As a result, you can speed up your development workflows and save on warehouse compute costs. Let's run a few commands using deferral:
```

File: website/docs/guides/qs-cloud-cli.md [hunk 6]
```diff
@@ -0,0 +1,313 @@
...
+
+    ```shell
+    dbt sqlfluff fix models/staging/campaigns/stg_campaigns.sql --dialect snowflake
+    ```
+
+    This attempts to directly make fixes in the `stg_campaigns` model.
+
+### Change branches
+
+You can quickly change branches without fully pushing to your git provider (such as GitHub):
```
Comment (nghi-ly) [inline website/docs/guides/qs-cloud-cli.md]:
```suggestion
You can quickly change branches without fully pushing to your Git provider (such as GitHub):
```

**Comment by nghi-ly** [comment]
log: accidentally hit the _green_ merge button before approving first. it was already approved in my mind (ie already in an approved state).

**Pull request merged**